code/modules/food_and_drink/plants.dm

```diff
@@ -1050,6 +1069,8 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 	slice_product = /obj/item/reagent_containers/food/snacks/plant/lime/wedge
 	slice_amount = 6
 	slice_suffix = "wedge"
+	brew_result = "limeade"
+	brew_amount = 30
```

Comment:
Why does this result in a higher yield than other products?

Reply:
Because wedges were brewable, and I wanted it to make sense that 6 wedges give 5u each to a combined 30u.